app/assets/javascripts/discourse/controllers/password-reset.js.es6

```diff
@@ -9,7 +9,7 @@ export default Ember.Controller.extend(PasswordValidation, {
   isDeveloper: Ember.computed.alias("model.is_developer"),
   admin: Ember.computed.alias("model.admin"),
   secondFactorRequired: Ember.computed.alias("model.second_factor_required"),
-  backupEnabled: Ember.computed.alias("model.second_factor_backup_enabled"),
+  backupEnabled: Ember.computed.alias("model.backup_enabled"),
```

Comment:
Was this changed?

Reply:
No, this was wrong and not used until now (it comes from here https://github.com/discourse/discourse/blob/master/app/controllers/users_controller.rb#L528)